### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by Electroso** [alsbi/HHC-N8I8OP#2]
Hello, 
I'm trying to get the input feedback but when I trigger the pushbutton I get it like 3 times:
Triggering input 8:
input10000000input00000000input10000000input00000000input10000000input00000000

Do you know what's happening?

**Comment by alsbi** [comment]
Hello,
button have jitter protection? I have "normally open" type button: 1 click - 1 response, default switch not suitable for physical input.

My HA module use relay in udp server mode, feedback update read from "read" udp command

**Comment by Electroso** [comment]
Where can I change the jitter protection? I don't see it in the software.

**Comment by alsbi** [comment]
It’s hardware button) 

how do you trigger push button? How was the relay configured before testing?

**Comment by Electroso** [comment]
I have a singlepole pushbutton and I want to get when it's pressed, so the input mode is Automatic and I receive the command:
input10000000input00000000input10000000input00000000input10000000input00000000
I don't need the relay I just want to receive the input change and then I will decide which relay to act.

**Comment by alsbi** [comment]
I don't think this relay is the best idea for such a task. I have not tested this behavior.

Probably in this mode it scans the state and should be parsed in real time, but it looks like the button itself is not suitable and gives multiple triggers. For physical input, you need a special button, with a click

**Comment by Electroso** [comment]
I'm doing it with the "special" button, but I still get the command wrong.
